### DISCUSSION TITLE
Github Student not authenticated

### DISCUSSION THREAD

**Discussion by sashimi3433** [community/community#121468] [GitHub Education]
### Hi everyone,

I am receiving the following error.

What happened? The item you uploaded is insufficient to prove your current academic status, please see suggestions. Suggested proof to expedite your approval:

Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you only have one legal name, enter it in both the first and last name fields of your GitHub billing information.

Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are not acceptable.

You do not appear to be located near a campus of your chosen school. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow remote access. You must not use a VPN. If you believe that our information about your school is incomplete or incorrect, such as missing campus location(s), please select "My selected school has incorrect or incomplete information e.g. domains or campus location" when submitting a GitHub Education support ticket.

I have my billing information and student ID information together. And the reason I am so far away from the school is because I attend a correspondence school. But there is no way to communicate that to the administration.

**Comment by samus-aran** [comment]
Hi @sashimi3433 

Does your correspondence school state that the campus you are attended is associated with it? You can request that the school details be updated to reflect that.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional school campuses.

Hope this helps! Let us know how you get along 👍

**Reply by Hopvine15** [reply]
Do you know how long on average it takes for the selected school incorrect information? 
As I raised a ticket a month ago now and have still yet to hear anything
